tests/store/crud/read/test_read_application.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+
+from cg.models.orders.constants import OrderType
+from cg.store.exc import EntryNotFoundError
+from cg.store.models import Application
+from cg.store.store import Store
+
+
+def test_get_active_applications_by_order_type_no_application(store: Store):
```

Comment:
Might be nice with a test for where there is an application to return

Reply:
I had one, I realised it is exactly the same test as `test_filter_applications_by_order_type` so I removed it

Reply:
fair